### DISCUSSION TITLE
Using Custom providers like Groq with Zed Ai

### DISCUSSION THREAD

**Discussion by giscus** [FarisZR/FarisZR.com#11] [Comments]
# Using Custom providers like Groq with Zed Ai

a quick guide on how to integrate custom Large Language Models into Zed, a new open-source code editor, using providers like Groq.

https://fariszr.com/use-custom-providers-with-zed-groq/

**Comment by jeffscottward** [comment]
this is stupid because its listing open ai models not whats coming back from Groq
